src/Microsoft.ML.Transforms/Text/WordEmbeddingsExtractor.cs

```diff
@@ -850,6 +833,27 @@ internal WordEmbeddingsExtractingEstimator(IHostEnvironment env, string customMo
             _columns = columns;
         }
 
+        /// <summary>
+        /// Information for each column pair.
+        /// </summary>
+        public sealed class ColumnInfo
+        {
+            public readonly string Name;
+            public readonly string InputColumnName;
+
+            public ColumnInfo(string name, string inputColumnName = null)
```

Comment:
summary to constructor.

Reply:
fixed

---
In reply to: [254031689](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254031689) [](ancestors = 254031689)